test/fixtures/manifest-stream.json

```diff
@@ -62,5 +62,14 @@
         "signature": "3045022100A4622D77AF27AD6E92166D8A4BC2B8A80646757C1F4F1E33F3C4BC9344765D38022062FC193AF1555E7D1D46B6023861ECD1415B319C488DFCB99648D99060A41120",
         "signing_key": "n9K7fyu8uvmCoWvW4ZQVCWgW2zrz7sh33Ao7ceNkL7iQGDYtuwTU",
         "type": "manifestReceived"
+    },
+    {
+        "manifest":"24000000017121ED3D6590743B5CCCB4005D07F34D4AC38D69BC052C281E63E2124E2E214E5EBFAA732103823768783B6C72E8B88B5DAE0FC684C0BA3C755EBA5A38F55A0D755C3B26E35D76463044022047E02FD193E1C0051B7951F6E2E8CEF3F1DC74549A229C42F4B9F45B54078F16022025B0533FBA16DC9F02CBBCBF346094A9230857DFAAA6126936EA6DCD6D8717B0701240DA58D1BF86B56829E5C25921331CEBB59421094967B631B5975885FE7F8CEF0A4908CE0318417D454E3CC76926C456CB0529280E7EE0CA1BD178D118CC53290D",
```

Comment:
So this has been added as a part of the manifest stream?

Reply:
Yeah I got an unhandled promise error from `handleManifest` in the VHS and this is the one that caused it.